parlai/tasks/ms_marco/build.py

```diff
@@ -9,22 +9,34 @@
 import gzip
 import json
 import os
+import pandas as pd
 
 import parlai.core.build_data as build_data
 
 
-def read_gz(filename, delete_gz=True):
-    f = gzip.open(filename, 'rb')
-    lines = [x.decode('utf-8') for x in f.readlines()]
-    if delete_gz:
-        os.remove(filename)
+def read_file(filename):
+    print("reached line loading")
+    with open(filename) as f:
+        lines = [x for x in f.readlines()]
     return lines
 
+def convert_file(input_file_path, output_file_path):
```

Comment:
Yes, please get rid of the dependency on pandas. It looks like it could be replaced with roughly:

```
records = json.read(f)
#...
for row in records:
    f.write(json.dumps(row))
    f.write('\n')
```